examples/src/main/java/org/apache/spark/examples/sql/JavaSparkSQL.java

```diff
@@ -133,7 +133,7 @@ public String call(Row row) {
     // Register this DataFrame as a table.
     peopleFromJsonFile.registerTempTable("people");
 
-    // SQL statements can be run by using the sql methods provided by sqlContext.
+    // SQL statements can be run by using the sql methods provided by spark (SparkSession object).
```

Comment:
I am less sure if `spark (SparkSession object)` is right. It was a variable name (`sqlContext)` but for now I add `(SparkSession object)` with the variable name because it seems the name `spark` is confusing. Feel free to correct this here.